store/interfaces.json

```diff
@@ -1,5 +1,6 @@
 {
   "order-placed": {
-    "component": "OrderPlaced"
+    "component": "OrderPlaced",
+    "allowed":["shop-review-form"]
```

Comment:
nit ⛏ 

```suggestion
    "allowed": ["shop-review-form"]
```

Reply:
I think you don't need that since you already let the user add via `store.order-placed`, right?